modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+
+. Click *Save*.
++
+As a result, the {gitops-title} Operator enables the namespace-scoped Argo Rollouts installation to reconcile Argo Rollouts CR.
+
+. Verify if the {gitops-title} Operator has enabled namespace-scoped Argo Rollouts installation by viewing the logs of the {gitops-shortname} container:
+.. In the *Administrator* perspective of the web console, go to *Workloads* -> *Pods*.
+.. Click the `openshift-gitops-operator-controller-manager` pod, and then click the *Logs* tab.
+.. Look for the following log statement: `Running in namespaced-scoped mode`. This indicates that the {gitops-title} Operator has enabled the namespace-scoped Argo Rollouts installation.
+
+. Create a `RolloutManager` resource to perform the namespace-scoped Argo Rollout installation:
+.. Go to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps*, and click the *RolloutManager* tab.
+.. Click the *Create RolloutManager* button.
+.. Select *YAML view* and enter the following snippet:
++
+.Example RolloutManager CR for namespace-scoped Argo Rollouts installation
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: RolloutManager
+metadata:
+  name: rollout-manager
+  namespace: my-application #<1> 
+spec:
+  namespaceScoped: true
+----
+<1> Indicate the name of the project where you want to perform the namespace-scoped Argo Rollout installation.
++
+.. Click *Create*. 
++
+After the `RolloutManager` CR is created, the {gitops-title} begins the namespace-scoped Argo Rollouts installation into the selected namespace.
+
+. Verify the successful namespace-scoped installation.  
+.. In the *RolloutManager* tab, under the *RolloutManagers* section, ensure that the *Status* field of the `RolloutManager` instance is `Phase: Available`.
+.. Examine the following output in the *YAML* tab under the *RolloutManagers* section to ensure that the installation is successful:
++
+.Example of namespace-scoped Argo Rollouts installation YAML file
+[source,yaml]
+----
+spec:
+  namespaceScoped: true
+status:
+  conditions:
+    lastTransitionTime: '2024-07-10T14:20:5z`
+    message: ''
+    reason: Success
+    status: 'True' #<1>
+    type: 'Reconciled'
+  phase: Available
+  rolloutController: Available
+----
+<1> This indicates that the namespace-scoped Argo Rollouts installation is enabled successfully.
++
+You can see an error message if a namespace-specific Argo Rollouts installation is attempted while a cluster-scoped installation already exists on the cluster:
++
+.Example of an incorrect installation with an error message
+[source,yaml]
+----
+spec:
+  namespaceScoped: false
+status:
+  conditions:
+   lastTransitionTime: '2024-07-10T14:10:7z`
+   message: 'when Subscription has environment variable Namespace_Scoped_Argo_RolloutsManagers: In this case, only namespace-scoped RolloutManager resources are supported'
```

Comment:
You can change the `message:` line to `message: when Subscription has environment variable NAMESPACE_SCOPED_ARGO_ROLLOUTS set to False, there may not exist any namespace-scoped RolloutManagers: only a single cluster-scoped RolloutManager is supported`

Here is the line in the code where that message is coming from, for reference: https://github.com/argoproj-labs/argo-rollouts-manager/blob/192dd2c3b5dd026e2c59c5765e98ca2f70ca01f9/controllers/utils.go#L22C46-L22C248

Reply:
Fixed.